.changeset/curvy-walls-kneel.md

```diff
@@ -0,0 +1,47 @@
+---
+'astro': patch
+---
+
+Adds support for allowing integrations to refresh the content layer
+
+This adds a new object to the `astro:server:setup` hook that allows integrations to refresh the content layer. This can be used for example to register a webhook endpoint during dev, or to open a socket to a CMS to listen for changes.
```

Comment:
```suggestion
Adds a new object `refreshContent` to the `astro:server:setup` hook that allows integrations to refresh the content layer. This can be used, for example, to register a webhook endpoint during dev, or to open a socket to a CMS to listen for changes.
```
My preference is always to get the new thing named asap, so we could do something like this!